source/scpd/types/Stellar_types.d

```diff
@@ -51,7 +51,7 @@ enum SignerKeyType : int32_t {
 /// Modified to suits scpd's needs
 /// Note: should only be used within this package
 package(scpd)
-struct PublicKey {
+extern(C++, class) struct PublicKey {
```

Comment:
Why?

Reply:
Incorrectly applied because it should only be used within spcd.